.github/workflows/pre_commit.yml

```diff
@@ -0,0 +1,28 @@
+name: add cherry-pick version labels to pre-commit-ci pr's
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - ready_for_review
+      - reopened
+      - synchronize
+
+
+jobs:
+  add-labels:
+    name: apply cherry-pick version labels.
+    if: github.actor == 'pre-commit-bot'
+    runs-on: ubuntu-latest
+    steps:
+      - name: apply supported version labels to cherry-pick PRs created by the pre-commit-bot.
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.CHERRYPICK_PAT }}
+          script: |
+            github.rest.issues.addLabels({
+              issue_number: ${{ steps.cherrypick.outputs.number }},
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              labels: ['CherryPick', '6.14.z', '6.13.z', '6.12.z']
```

Comment:
Not sure, if its possible to define supported branches and it's label somewhere so we don't hardcode it everytime, wdyt @omkarkhatavkar ?

Reply:
I thought about it, I had something in my mind let me try out that.

Reply:
@omkarkhatavkar Did you get a chance to try this out?

Reply:
@Gauravtalreja1 I know it is been bit late, can you please take a review here ?